### PR TITLE
CB-13930: removed deprecated platforms and plugins. Removed old apach…

### PR DESCRIPTION
**Pull request by stevengill**
…e links

### Checklist
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

File: www/_data/fetched-files.yml [hunk 1]
```diff
@@ -29,41 +29,13 @@
         commit: "master"
     dest:
         path: "reference/cordova-plugin-camera/index.md"
--
-    src:
-        repoName: "apache/cordova-plugin-console"
-        packageName: "cordova-plugin-console"
-        commit: "master"
-    dest:
-        path: "reference/cordova-plugin-console/index.md"
```
Comment (dblotsky) [inline www/_data/fetched-files.yml]: Should this one have gotten removed?

File: www/_data/toc/en_dev-src.yml [hunk 2]
```diff
@@ -55,22 +50,16 @@
             children:
                 - url: reference/cordova-plugin-battery-status/index.html
                 - url: reference/cordova-plugin-camera/index.html
-                - url: reference/cordova-plugin-console/index.html
```
Comment (dblotsky) [inline www/_data/toc/en_dev-src.yml]: Should this one also have gotten removed?

File: www/_data/contribute-repo-links.yml [hunk 3]
```diff
@@ -1,177 +1,102 @@
 -   title: Platforms
     projects:
-    -   project: Amazon Fire OS
-        apache: https://git-wip-us.apache.org/repos/asf?p=cordova-amazon-fireos.git;a=summary
-        mirror: https://github.com/apache/cordova-amazon-fireos
 
     -   project: Android
-        apache: https://git-wip-us.apache.org/repos/asf?p=cordova-android.git;a=summary
```
Comment (janpio) [inline www/_data/contribute-repo-links.yml]: Where was this used?

File: www/_data/fetched-files.yml (hunk 1 above)
Comment (janpio) [reply www/_data/fetched-files.yml]: Yes, I think so as https://github.com/apache/cordova-plugin-console mentions it is deprecated and included in the platforms directly.

File: www/_data/contribute-repo-links.yml (hunk 3 above)
Comment (janpio) [reply www/_data/contribute-repo-links.yml]: Ah, found the change below.

**Comment by janpio** [comment]
Looking good.

**Comment by stevengill** [comment]
@dblotsky the console plugin got integrated into ios. For cordova@8, if you install it in your project, it will just skip installing it due to the engine requirements. It is still useful for older ios versions but that won't work with cordova@8. So it will stay in the docs for older versions, but removed in the newer ones.

File: www/contribute/index.html [hunk 4]
```diff
@@ -55,7 +55,7 @@ <h2>{{ projectType.title }}</h2>
             {% for project in projectType.projects %}
 
             {% if firstProjectType and forloop.first %}
-                {% assign itemClass = "first" %}
+                {% assign itemClass = "" %}
```
[the remark this answers is not in the document]
Comment (stevengill) [reply www/contribute/index.html]: I just did it as part of removing the apache links. Added it back in

**Comment by stevengill** [comment]
@dblotsky so about translations, we have recently been discussing on the list of dropping all translations. Here is an issue for it https://issues.apache.org/jira/browse/CB-13931

Here is the mailing list thread http://apache.markmail.org/thread/fujkivnzx5jyzjaj

**Comment by stevengill** [comment]
Finally, plugin docs will have to be eventually updated dropping old platforms. 

I've made the changes suggested here. Take a final look, but I think this is good to merge

**Comment by stevengill** [comment]
Okay, I'm merging this in but we can continue discussing.

**Pull request merged**